dashboard/public/js/main.js

```diff
@@ -20,12 +20,12 @@ html5indexedDB.db = null;
 var filestoreName = 'sugar_filestore';
```

Comment:
These two libraries are imported:
- dashboard/public/js/jquery.datetimepicker.min.js
- dashboard/public/js/jquery.datetimepicker.full.js

Only one of them is needed.

Reply:
Imports aren't present in this file, one of the files is imported in the footer.ejs , extra file I'm deleting